install_plugin.yml

```diff
@@ -0,0 +1,4 @@
+import: install.yml
+
+install:
+  - "npm install @hapi/hapi@19"
```

Comment:
This will only test on hapi 19. Does it work to create an `env.yml` like [this](https://github.com/hapijs/basic/blob/3369f0a7df7ae1fab7fec642cb83dbd2cd7ceb57/.travis.yml#L13-L15) (except I'd change to v19 and v20). Then, in this file, install `@hapi/hapi@$HAPI_VERSION`.

Reply:
The `import: install.yml` will also merge the `npm install` from that file so it'll test with the latest version and v19.

Reply:
Using [`basic`](https://github.com/hapijs/basic/blob/3369f0a7df7ae1fab7fec642cb83dbd2cd7ceb57/.travis.yml#L9-L11) as an example: I think it will run `npm install` followed by `npm install @hapi/hapi@$HAPI_VERSION` before running the tests. If that's correct, then this will only ever run the tests on hapi 19. I'm happy to be wrong though.

Reply:
I believe you are right. I skimmed through the Travis doc but was unable to find the exhaustive list of the matrix expansion keys. It would make sense that `install` is not one of them thus the need of using `env` which is a matrix expansion key.

I think you just need to update the last `npm install` to use an env var as you suggested and use an `env` property to set that var.